WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsItem.swift

```diff
@@ -0,0 +1,201 @@
+                            .foregroundColor(.withColorStudio(name: .red, shade: .shade60))
+                            .footnoteStyle()
+                            .renderedIf(viewModel.valuePerUnit.isEmpty)
+                    }
+
+                    VStack(alignment: .leading) {
+                        Text(Localization.weightPerUnitTitle)
+                            .foregroundColor(.primary)
+                            .subheadlineStyle()
+                        HStack {
+                            TextField("0", text: $viewModel.weightPerUnit)
+                                .padding(Layout.extraPadding)
+                            // TODO: Add right unit
+                            Text("kg")
+                                .font(.subheadline)
+                                .foregroundStyle(.secondary)
+                                .padding(.trailing, Layout.unitsHorizontalSpacing)
+                        }
+                        .roundedBorder(cornerRadius: Layout.borderCornerRadius,
+                                       lineColor: viewModel.weightPerUnit.isEmpty ? .withColorStudio(name: .red, shade: .shade60) : Color(.separator),
+                                       lineWidth: Layout.borderLineWidth)
+                        Text(Localization.valueRequiredWarningText)
+                            .foregroundColor(.withColorStudio(name: .red, shade: .shade60))
+                            .footnoteStyle()
+                            .renderedIf(viewModel.weightPerUnit.isEmpty)
+                    }
+                }
+
+                HStack {
+                    Text(Localization.originCountryTitle)
+                        .foregroundColor(.primary)
+                    Spacer()
+                    Button {
+                        // TODO: Add information
+                    } label: {
+                        Image(systemName: "info.circle")
+                            .foregroundColor(Color(.wooCommercePurple(.shade60)))
+
+                    }
+                }
+                    .subheadlineStyle()
+
+                Button {
+                    isShowingCountries = true
+                } label: {
+                    HStack {
+                        Text(viewModel.originCountry.name)
+                            .bodyStyle()
+                        Spacer()
+                        Image(systemName: "chevron.up.chevron.down")
+                    }
+                    .padding()
+                }
+                .roundedBorder(cornerRadius: Layout.borderCornerRadius, lineColor: Color(.separator), lineWidth: Layout.borderLineWidth)
+            }
+            .padding(.leading, Layout.extraPadding)
+            .padding(.trailing, Layout.extraPadding)
+            .padding(.bottom, Layout.extraPadding)
+        })
+        .roundedBorder(cornerRadius: Layout.borderCornerRadius, lineColor: Color(.separator), lineWidth: Layout.borderLineWidth)
```

Comment:
According to the design, the border around the collapsible view could be grey (when collapsed), red (when collapsed if there's an issue with the item), or purple (when expanded).

Reply:
Good catch! Done in 121550512ec4446e580253c595cccfbbb1c38346